package.json

```diff
@@ -38,6 +40,7 @@
         "gatsby-plugin-react-helmet": "^5.24.0",
         "gatsby-plugin-react-helmet-canonical-urls": "^1.4.0",
         "gatsby-plugin-robots-txt": "^1.7.1",
+        "gatsby-plugin-rudderstack": "^2.0.0",
```

Comment:
I'm not sure if we need this, because the SDK in the `@deriv/analytics` package would handle all of the plugin's described APIs

Reply:
Nice one @adrienne-deriv I've removed it now. Let's see if it builds.